test/benchee/conversion/count_test.exs

```diff
@@ -1,58 +1,63 @@
 defmodule Benchee.Conversion.CountTest do
   use ExUnit.Case
+  alias Benchee.Conversion.Count
   import Benchee.Conversion.Count
   doctest Benchee.Conversion.Count
 
+  defp unit_for(unit_atom) do
+    Map.fetch! Count.units, unit_atom
+  end
```

Comment:
What if we added functions to `Count` so instead of `unit_for`, we could do `Count.billion` ?